tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2060,4 +2060,57 @@ public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $
 			)
 		);
 	}
+
+	/**
+	 * @Given /^the administrator "([^"]*)" has added the following users to a group "([^"]*)" at once using the Graph API$/
+	 *
+	 * @param string $user
+	 * @param string $group
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function theAdministratorHasAddedTheFollowingUsersToAGroupAtOnceUsingTheGraphApi(string $user, string $group, TableNode $table) {
+		$userIds = [];
+		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");
+		foreach ($table->getHash() as $row) {
+			$userIds[] = $this->featureContext->getAttributeOfCreatedUser($row['username'], "id");
+		}
+		$this->addMultipleUsersToGroup($user, $userIds, $groupId, $table);
+		$response = $this->featureContext->getResponse();
+		if ($response->getStatusCode() !== 204) {
+			$this->throwHttpException($response, "Could add users in group'$group'");
+		}
+		$this->featureContext->emptyLastHTTPStatusCodesArray();
+	}
+
+	/**
+	 * @When /^the administrator "([^"]*)" tries to add a group "([^"]*)" to a group "([^"]*)" at once using the Graph API$/
+	 *
+	 * @param string $user
+	 * @param string $groupToAdd
+	 * @param string $group
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function theAdministratorAddGroupToAGroupAtOnceUsingTheGraphApi(string $user, string $groupToAdd, string $group) {
+		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");
+		$userId = $this->featureContext->getAttributeOfCreatedGroup($groupToAdd, "id");
```

Comment:
`$userId` seems confusing. Rename it to like `groupToAddId` or something else.

Reply:
i also got confused which name suit best $userId or $groupIdTo Add